src/private/MDMPerformerGroup.h

```diff
@@ -37,7 +37,7 @@
 // nil by default. Useful for view duplication.
 @property(nonatomic, nullable) id schedulerTarget;
 
-- (void)addPlan:(nonnull id<MDMPlan>)plan trace:(nonnull MDMTrace *)trace;
+- (void)addPlan:(nonnull id<MDMPlan>)plan trace:(nonnull MDMTrace *)trace log:(nullable MDMTransactionLog *)log;
```

Comment:
Added the following methods to `MDMPerformerGroup`:
```
- (void)addPlan:(nonnull id<MDMPlan>)plan to:(nonnull id)target trace:(nonnull MDMTrace *)trace;
- (void)addPlan:(nonnull id<MDMNamedPlan>)plan named:(nonnull NSString *)name to:(nonnull id)target trace:(nonnull MDMTrace *)trace;
- (void)removePlanNamed:(nonnull NSString *)name from:(nonnull id)target;
```
which is helping to split up this logic a little.